### ISSUE TITLE
wsarecv an existing connection was forcibly closed by the remote host 

### ISSUE THREAD

**Issue by imal987** [gophish/gophish#918]
What version of Gophish are you using?: 0.4

Hello!! I'm using gophish on windows server 2012 r2 in vmware workstation environment!! when I open the admin page and want to start creating sending profile nothing happen... I mean when I click on the button new profile no page appears nothing happen!! when i return to the cmd and see I get this following message " **wsarecv an existing connection was forcibly closed by the remote host** " Any help on this please !!

Thanks a lot :)

**Comment by jordan-wright** [comment]
Hi there!

What browser are you using? Have you checked the developer tools console in the browser for any errors? Does this happen only when creating a sending profile? Are you sure this occurs exactly when you press the button?

Also, are you seeing this every time? If so, you might try Googling "wsarecv an existing connection was forcibly closed by the remote host" and checking out the other issues that pop up. Some people report having various other software that might cause some unintended effects.

**Comment by imal987** [comment]
Hello, thanks for your response!!  
What browser are you using? : Internet explorer 
Have you checked the developer tools console in the browser for any errors? : yes I checked but there is no specified errors
Does this happen only when creating a sending profile? : NO, it happens even when I click to create groups or create Template or Create landing page even when I click on UI settings nothing happen I can surf just on the main parts but when it comes to start to create something nothing appear!!! 
Are you sure this occurs exactly when you press the button? : I'm not sure about the exactly thing but I think it's related to it  because when I open the cmd exactly after I click the button I see the error!! 

I tried to Google it but I didn't reach any possible solution that's why I came here !!

Thanks !!
